pkg/controller/direct/apikeys/apikeyskey_test.go

```diff
@@ -0,0 +1,152 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apikeys
+
+import (
+	"context"
+	"encoding/json"
+	"strings"
+	"testing"
+
+	pb "cloud.google.com/go/apikeys/apiv2/apikeyspb"
+	"github.com/GoogleCloudPlatform/k8s-config-connector/pkg/clients/generated/apis/apikeys/v1alpha1"
+	"github.com/google/go-cmp/cmp"
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/testing/protocmp"
+	"sigs.k8s.io/yaml"
+)
+
+func TestBuildCreate(t *testing.T) {
+	ctx := context.TODO()
```

Comment:
Unless we believe that TestBuildCreate will add a context parameter this should probably be just context.Background()

Reply:
Replaced with `ctx := test.WithContext(context.Background(), t)`